rosettanet/2.0/modules/ROOT/pages/rosettanet-connector-examples.adoc

```diff
@@ -0,0 +1,486 @@
+	</http:listener-config>
+	<http:request-config name="HTTP_Request_configuration"
+	doc:name="HTTP Request configuration" >
+		<http:request-connection host="localhost" port="8081" />
+	</http:request-config>
+	<rosetta:config name="PO_ResponderConfig_Seller"
+	pipRole="RESPONDER"
+	doc:name="RosettaNet Config"
+	listenerConfigName="HTTP_Listener_config"
+	servicePath="/partner2"
+	requesterConfigName="HTTP_Request_configuration"
+	requestPath="/partner1"
+	keystorePath="/partner2.jks"
+	keystorePass="nosecret"
+	privatePass="partner2"
+	globalUsageCode="Test"
+	partnerBusinessIdentifier="123456789"
+	partnerLocationId="partner1"
+	selfBusinessIdentifier="123456788"
+	selfLocationId="partner2" pipFile="/PIP3A4/V02_02.xml"/>
+	<http:listener-config name="HTTP_InputListener"
+	doc:name="HTTP Listener config" >
+		<http:listener-connection host="localhost" port="8802" />
+	</http:listener-config>
+	<flow name="Send-Purchase-Order-Confirmation">
+		<http:listener doc:name="Listener"
+		config-ref="HTTP_InputListener"
+		path="/" allowedMethods="POST"/>
+		<rosetta:send-action doc:name="Send action"
+		 config-ref="PO_ResponderConfig_Seller"/>
+		<ee:transform doc:name="Transform Message">
+			<ee:message >
+				<ee:set-payload ><![CDATA[%dw 2.0
+output text/plain
+---
+"Seller sent action message " ++ attributes.messageId ++ "\n"]]></ee:set-payload>
+			</ee:message>
+		</ee:transform>
+	</flow>
+	<flow name="Receive-Purchase-Order-Request">
+		<rosetta:action-source doc:name="Action source"
+		config-ref="PO_ResponderConfig_Seller"/>
+		<logger level="INFO" doc:name="Logger"
+		 message="Seller received action message #[attributes.messageId]"/>
+	</flow>
+	<flow name="Send-Purchase-Order-Completion">
+		<rosetta:completion-source doc:name="Completion"
+		config-ref="PO_ResponderConfig_Seller"/>
+		<logger level="INFO" doc:name="Logger"
+		message="Seller action message #[attributes.replyAttributes.messageId] completed as #[payload.completionCode]"/>
+	</flow>
+</mule>
+----
+
+== Test the Examples
+
+To test the Buyer and Seller apps, do the following after you run the apps:
+
+. Provide the purchase order request to the Buyer app.
+. Provide the purchase order request response to the Seller app.
+
+=== Provide the Purchase Order Request Document
+
+Use an HTTP POST to the Buyer app `HTTP_InputListener` endpoint to provide the purchase order request to send to the seller.
+
+You can download a https://s3-us-west-2.amazonaws.com/mulesoft-sites-vendorcontent/public-assets/sample-purchase-order-request-content.xml[sample purchase order request]. You can then use any HTTP tool, such as a browser plugin, standalone tool such as PostMan, console tool such as `curl` to POST the data to the Buyer app.
+
+For example, the following `curl` command posts a purchase order request:
+
+`+curl -v -H "Content-Type: application/text" -XPOST --data-binary @sample-purchase-order-request-content.xml http://localhost:8801+`
```

Comment:
Is the port correct? It's usually 8081. Although, it looks like maybe this example is using 8082?

Reply:
Correct as is according to the dev.